freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/model/UserSyncOptions.java

```diff
@@ -0,0 +1,28 @@
+package com.sequenceiq.freeipa.service.freeipa.user.model;
+
+public class UserSyncOptions {
+
+    private final boolean fullSync;
+
+    private final boolean fmsToFreeIpaBatchCallEnabled;
+
+    private final boolean credentialsUpdateOptimizationEnabled;
+
+    public UserSyncOptions(boolean fullSync, boolean fmsToFreeIpaBatchCallEnabled, boolean credentialsUpdateOptimizationEnabled) {
+        this.fullSync = fullSync;
+        this.fmsToFreeIpaBatchCallEnabled = fmsToFreeIpaBatchCallEnabled;
+        this.credentialsUpdateOptimizationEnabled = credentialsUpdateOptimizationEnabled;
+    }
+
+    public boolean fullSync() {
+        return fullSync;
+    }
+
+    public boolean fmsToFreeIpaBatchCallEnabled() {
+        return fmsToFreeIpaBatchCallEnabled;
+    }
+
+    public boolean credentialsUpdateOptimizationEnabled() {
+        return credentialsUpdateOptimizationEnabled;
+    }
```

Comment:
please start the method name with a verb if possible. in case of booleans it's usually `is` eg:
`isFullSync`

Reply:
Done.